packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -430,14 +428,28 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
     this.forceUpdate();
   }
 
+  /**
+   * Only to be called when there is nothing in the input. Checks to see if the consumer has
+   * provided a function to resolve suggestions
+   */
   protected onEmptyInputFocus() {
     const emptyResolveSuggestions = this.props.onEmptyResolveSuggestions
       ? this.props.onEmptyResolveSuggestions
       : // tslint:disable-next-line:deprecation
         this.props.onEmptyInputFocus;
 
-    const suggestions = emptyResolveSuggestions!(this.state.items);
-    this.updateSuggestionsList(suggestions);
+    // Only attempt to resolve suggestions if it exists
+    if (emptyResolveSuggestions) {
+      const suggestions = emptyResolveSuggestions!(this.state.items);
+
+      this.updateSuggestionsList(suggestions);
+
+      this.setState({
```

Comment:
This seems to be new logic and not just a refactor. Intentional?

Reply:
Correct! This fixes an issue where a default value in the picker wouldn't have triggered a resolve suggestions.